setup.py

```diff
@@ -27,7 +27,7 @@
     name='cimpyorm',  # Required
     # https://www.python.org/dev/peps/pep-0440/
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.9.4',  # Required
+    version='0.9.5',  # Required
```

Comment:
```suggestion
    version='0.10.0',  # Required
```

If we drop support for 3.7 and potentially 3.8.

Reply:
Sounds like a good idea. I removed 3.8 from the CI-configs and set the version to 0.10.0